package.json

```diff
@@ -3559,23 +3559,23 @@
                 "command": "aws.codeWhisperer",
                 "key": "alt+c",
                 "mac": "alt+c",
-                "when": "editorTextFocus && CODEWHISPERER_ENABLED"
+                "when": "editorTextFocus && CODEWHISPERER_ENABLED || editorTextFocus && isCloud9"
             },
             {
                 "command": "aws.codeWhisperer.rejectCodeSuggestion",
                 "key": "escape",
                 "mac": "escape",
-                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED"
+                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED || inlineSuggestionVisible && !editorReadonly && isCloud9"
             },
             {
                 "key": "right",
                 "command": "editor.action.inlineSuggest.showNext",
-                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED"
+                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED || inlineSuggestionVisible && !editorReadonly && isCloud9"
             },
             {
                 "key": "left",
                 "command": "editor.action.inlineSuggest.showPrevious",
-                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED"
+                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED || inlineSuggestionVisible && !editorReadonly && isCloud9"
             }
```

Comment:
ahhh, I feel I don't quite understand these 2 commands actually given we are not providing our own command, do we need the changes here?

Reply:
I actually don't know how to test left/right in cloud9. I only tested alt+c

Reply:
For simplicity we could leave these out, and only change alt+c for now.